### PR TITLE
Allows Sidekiq configuration per queue & allows filtering of providers on the check end-point

### PR DESCRIPTION
**Pull request by danielnc**
* Fixed tests failing for rails 5.2
* added monitoring for different sidekiq queues
* added a param to filter which checks to run on the check end-point
* bump gem version

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,11 +4,13 @@ rvm:
   - 2.3.6
   - 2.4.3
   - 2.5.0
+  - 2.5.1
```
Comment (lbeder) [inline .travis.yml]: You can just replace `2.5.0` with `2.5.1`.

File: README.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ This is a health monitoring Rails mountable plug-in, which checks various servic
 
 Mounting this gem will add a '/check' route to your application, which can be used for health monitoring the application and its various services. The method will return an appropriate HTTP status as well as an HTML/JSON/XML response representing the state of each provider.
 
+You can filter which checks to run by passing a parameter called providers.
```
Comment (lbeder) [inline README.md]: Can you change "providers" to "`providers`"?

File: README.md [hunk 3]
```diff
@@ -157,6 +210,14 @@ HealthMonitor.configure do |config|
     sidekiq_config.queue_size = 50
   end
 end
+
+# To configure specific queues
+HealthMonitor.configure do |config|
+  config.sidekiq.configure do |sidekiq_config|
+    sidekiq_config.add_queue_configuration("critical", latency: 10.seconds, size: 20)
```
Comment (lbeder) [inline README.md]: 👍 👍 👍

File: app/controllers/health_monitor/health_controller.rb [hunk 4]
```diff
@@ -25,7 +25,7 @@ def check
     private
 
     def statuses
-      res = HealthMonitor.check(request: request)
+      res = HealthMonitor.check(request: request, params: params)
```
Comment (lbeder) [inline app/controllers/health_monitor/health_controller.rb]: Can you make sure to sanitize params (e.g., using strong parameters) to only support `providers` for now and nothing else?

File: spec/controllers/health_monitor/health_controller_spec.rb [hunk 5]
```diff
@@ -50,6 +50,54 @@
...
+                  'status' => 'OK'
+                }
+              ],
+              'status' => 'ok',
+              'timestamp' => time.to_s(:rfc2822)
+            )
+          end
+        end
+
+        context 'single provider' do
```
Comment (lbeder) [inline spec/controllers/health_monitor/health_controller_spec.rb]: Can you also add a check of querying non-existing providers?

**Comment by danielnc** [comment]
@lbeder thanks for the review, addressed all comments on a new commit

**Comment by lbeder** [comment]
Well done!

**Pull request merged**